src/main/webapp/js/datepicker.js

```diff
@@ -89,7 +93,8 @@ function getMaxDateForVisibleDate(startDate, publishDate) {
 
 /**
  * @assumption: visibleDate has a valid value
- * @returns {Date} visibleDate
+ * @param {Date} visibleDate
+ * @returns {Date}
  */
 function getMinDateForPublishDate(visibleDate) {
     return visibleDate;
```

Comment:
@Neurrone The function is used in the `$(document).ready(function() {});` block. So I guess it is a kind of abstraction although it just return the input.